holoviews/tests/element/testtiles.py

```diff
@@ -0,0 +1,114 @@
+
+        lon, lat = Tiles.easting_northing_to_lon_lat(-2130123, 1829312)
+        self.assertAlmostEqual(lon, -19.1352205, places=2)
+        self.assertAlmostEqual(lat, 16.2122187, places=2)
+
+        lon, lat = Tiles.easting_northing_to_lon_lat(-1000000, 5000000)
+        self.assertAlmostEqual(lon, -8.9831528, places=2)
+        self.assertAlmostEqual(lat, 40.9162745, places=2)
+
+        lon, lat = Tiles.easting_northing_to_lon_lat(-20037508.34, 20037508.34)
+        self.assertAlmostEqual(lon, -180.0, places=2)
+        self.assertAlmostEqual(lat, 85.0511288, places=2)
+
+    def test_check_lonlat_to_eastingnorthing_identity(self):
+        for lon in np.linspace(-180, 180, 100):
+            for lat in np.linspace(-85, 85, 100):
+                easting, northing = Tiles.lon_lat_to_easting_northing(lon, lat)
+                new_lon, new_lat = Tiles.easting_northing_to_lon_lat(easting, northing)
+                self.assertAlmostEqual(lon, new_lon, places=2)
+                self.assertAlmostEqual(lat, new_lat, places=2)
+
+    def test_check_eastingnorthing_to_lonlat_identity(self):
+        for easting in np.linspace(-20037508.34, 20037508.34, 100):
+            for northing in np.linspace(-20037508.34, 20037508.34, 100):
+                lon, lat = Tiles.easting_northing_to_lon_lat(easting, northing)
+                new_easting, new_northing = Tiles.lon_lat_to_easting_northing(lon, lat)
+                self.assertAlmostEqual(easting, new_easting, places=2)
+                self.assertAlmostEqual(northing, new_northing, places=2)
+
+    def check_array_type_preserved(self, constructor, array_type, check):
+        lons, lats = np.meshgrid(
+            np.linspace(-180, 180, 100), np.linspace(-85, 85, 100)
+        )
+        lons = lons.flatten()
+        lats = lats.flatten()
+
+        array_lons = constructor(lons)
+        array_lats = constructor(lats)
+
+        self.assertIsInstance(array_lons, array_type)
+        self.assertIsInstance(array_lats, array_type)
+
+        eastings, northings = Tiles.lon_lat_to_easting_northing(
+            array_lons, array_lats
+        )
+        self.assertIsInstance(eastings, array_type)
+        self.assertIsInstance(northings, array_type)
+
+        new_lons, new_lats = Tiles.easting_northing_to_lon_lat(
+            eastings, northings
+        )
+        self.assertIsInstance(new_lons, array_type)
+        self.assertIsInstance(new_lats, array_type)
+
+        check(array_lons, new_lons)
+        check(array_lats, new_lats)
+
+    def test_check_numpy_array(self):
+        self.check_array_type_preserved(
+            np.array, np.ndarray,
+            lambda a, b: np.testing.assert_array_almost_equal(a, b, decimal=2)
+        )
+
+    def test_pandas_series(self):
+        self.check_array_type_preserved(
+            pd.Series, pd.Series,
+            lambda a, b: pd.testing.assert_series_equal(
+                a, b, atol=0.01, check_exact=False
```

Comment:
atol does not exist for the py2 pandas version it's getting.

Reply:
Sorry you still have to worry about py2 here!

Reply:
Ahh, thanks for the pointer!